e2e/test/scenarios/metrics/reproductions/metrics-reproductions.cy.spec.ts

```diff
@@ -61,3 +71,81 @@ describe("issue 47058", () => {
     });
   });
 });
+
+describe("issue 44171", () => {
+  const METRIC_A: StructuredQuestionDetails = {
+    name: "Metric 44171-A",
+    type: "metric",
+    display: "line",
+    query: {
+      "source-table": ORDERS_ID,
+      aggregation: [["count"]],
+      breakout: [
+        [
+          "field",
+          ORDERS.CREATED_AT,
+          { "temporal-unit": "month", "base-type": "type/DateTime" },
+        ],
+      ],
+    },
+  };
+
+  const METRIC_B: StructuredQuestionDetails = {
+    name: "Metric 44171-B",
+    type: "metric",
+    display: "line",
+    query: {
+      "source-table": ORDERS_ID,
+      aggregation: [["count"]],
+      breakout: [
+        [
+          "field",
+          ORDERS.CREATED_AT,
+          { "temporal-unit": "month", "base-type": "type/DateTime" },
+        ],
+      ],
+    },
+  };
+
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    createQuestion(METRIC_A);
+    createQuestion(METRIC_B).then(({ body: { id } }) => {
+      cy.visit(`/metric/${id}`);
+    });
```

Comment:
`createQuestion` accepts `visitQuestion` param but it does not support metrics yet.
Let's update that function to support metrics :pray: 

```suggestion
    createQuestion(METRIC_B, { visitQuestion: true });
```